### PR TITLE
chore: adding bug issue template

### PR DESCRIPTION
**Pull request by axel7083**
### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-extension-ai-lab/issues/959

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of the software are you running?
+      options:
+        - "next (development version)"
+        - "1.0.0"
+        - "0.4.0"
```
Comment (jeffmaury) [inline .github/ISSUE_TEMPLATE/bug_report.yml:50]: Remove versions previous to 1.0.0

**Review by jeffmaury — DISMISSED** [review]
Also add other types of issues as well

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (axel7083) [reply .github/ISSUE_TEMPLATE/bug_report.yml:50]: Done

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        - "Other"
+
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of the software are you running?
+      options:
+        - "next (development version)"
+        - "1.0.0"
```
Comment (benoitf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:49]: in Podman Desktop the release job automate the update of this file, how it will be updated there ?

**Review by benoitf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/epic.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      value: |
+        Epics are normally created by the development team, to group a set of related features and plan work across multiple sprints.
+        The features this epic includes are referenced with the text of the epic.
+
+  - type: textarea
+    id: domain
+    attributes:
+      label: Epic domain
+      description: A clear and concise description of the feature area or domain that this epic will address.
+      placeholder: Podman Desktop should support [...]
```
Comment (benoitf) [inline .github/ISSUE_TEMPLATE/epic.yml:17]: please update the title (not Podman Desktop)

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (axel7083) [reply .github/ISSUE_TEMPLATE/bug_report.yml:49]: ✨Manually ✨

File: .github/ISSUE_TEMPLATE/epic.yml (hunk 3 above)
Comment (axel7083) [inline .github/ISSUE_TEMPLATE/epic.yml:17]:
```suggestion
      placeholder: AI-Lab should support [...]
```

Comment (axel7083) [reply .github/ISSUE_TEMPLATE/epic.yml]: thx

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/epic.yml]: AFAIK It's `AI Lab` not `AI-Lab`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      description: "Which operating system are you on? Please provide the version as well. If you are on a Mac, please specify Apple silicon or Intel."
+      placeholder: "macOS Ventura 13.4 (Arm), Windows 11"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: install
+    attributes:
+      label: Installation Method
+      description: "How did you install AI-Lab ?"
```
Comment (axel7083) [inline .github/ISSUE_TEMPLATE/bug_report.yml:36]:
```suggestion
      description: "How did you install AI Lab ?"
```

**Review by lstocchi — APPROVED** [review]

**Comment by axel7083** [comment]
cc @jeffmaury

**Review by benoitf — APPROVED** [review]

**Pull request merged**